setup.py

```diff
@@ -10,7 +10,9 @@ def read(filename):
     filename = os.path.join(os.path.dirname(__file__), filename)
     text_type = type(u'')
     with io.open(filename, mode='r', encoding='utf-8') as fd:
-        return re.sub(text_type(r':[a-z]+:`~?(.*?)`'), text_type(r'``\1``'), fd.read())
```

Comment:
@19201080 one small thing: this changeset is not relevant to the PR, try always to be minimal and pertinent, so it's easy to track the intent of your contribution

Reply:
Oups my bad! i'll know for next time